js/core.js

```diff
@@ -271,11 +271,11 @@ var ERMrest = (function (module) {
                 // find alternative tables
                 // requires foreign keys built
                 for (s = 0; s < schemaNames.length; s++) {
-                    schema = self.schemas.get(schemaNames[s]);
-                    tables = schema.tables.names();
-                    for (t = 0; t < tables.length; t++) {
-                        table = schema.tables.get(tables[t]);
-                        table._findAlternatives();
+                    var schema1 = self.schemas.get(schemaNames[s]);
```

Comment:
Instead of renaming the variables, you can simply declare them before both the for loops start. For example, you could have declared `schema`, `tables` and `table` initially and then used it inside the for loop.

```js
var schema, table, tables
```